src/main/java/teammates/common/util/TimeHelper.java

```diff
@@ -499,17 +499,19 @@ public static LocalDateTime parseLocalDateTime(String dateTimeString, String pat
     }
 
     /**
-     * Parses a `LocalDateTime` object from separated date, hour and minute strings.
+     * Parses a {@code LocalDateTime} object for session form from separated date, hour and minute strings.
```

Comment:
I don't think `for session form` is necessary in this description?

Reply:
Oh I just read @darrenwee's comment. I think methods should not know where they are being called from. The format methods are slightly different; the location they are called from is used to indicate the format that is returned, i.e. "return this date in a format that is suitable for the sessions form", and not "this method is only to be called from the sessions form".

Reply:
Hence I'd also like to propose renaming the method back to its original name. @darrenwee thoughts?

Reply:
I think renaming it back to the original is not a good idea because the formatter string is hard coded into the method and acts as a facade for the actual `parseLocalDateTime(String, String)` method.

The `EEE, dd MMM...` format is only used in sessions-related forms, like the admin's sessions page, hence the original request to change the method name. If you had a method of name like `parseLocalDateTime(String date, String hour, String minute)`, the name would not tell you about what format it expects for each date/hour/minute field even though the pattern has been hard coded. The `format..` methods' names dictate what format the output will be suitable for, so the `parse...` methods (the inverse of the format methods) should also dictate what format type the input will be.

Reply:
That makes sense. I think the `For` should be changed to `From` though.